src/stacks-hierarchy/RootStack.tsx

```diff
@@ -168,6 +170,16 @@ export const RootStack = () => {
     };
   }
 
+  const isCurrentAppVersionLowerThanMinVersion =
+    compareVersion(APP_VERSION, minimum_app_version) > 0;
+
+  if (
+    APP_VERSION &&
+    minimum_app_version &&
```

Comment:
Tiny detail:
These two conditions could be moved inside `isCurrentAppVersionLowerThanMinVersion`, because the value of that variable isn't really valid unless we know the inputs are valid.

Reply:
Indeed, fixed now!